ddtrace/contrib/_httplib.py

```diff
@@ -8,15 +8,9 @@
 
 The httplib integration is disabled by default. It can be enabled when using
 :ref:`ddtrace-run<ddtracerun>` or :ref:`import ddtrace.auto<ddtraceauto>`
-using the ``DD_TRACE_HTTPLIB_ENABLED`` environment variable::
```

Comment:
Is `DD_TRACE_HTTPLIB_ENABLED` no longer recommended? Curious why this is being removed when it's patch_all being deprecated.

Reply:
Ah wait, sorry I forgot patch_all looks for that. Fixing. Good catch thank you!

Reply:
added back